src/components/elements/MapPolygonPanel/AttributeInformation.tsx

```diff
@@ -0,0 +1,152 @@
+import Dropdown from "@/components/elements/Inputs/Dropdown/Dropdown";
+import Input from "@/components/elements/Inputs/Input/Input";
+
+import Text from "../Text/Text";
+
+const dropdownOptionsRestoration = [
+  {
+    title: "Tree Planting",
+    value: 1
+  },
+  {
+    title: "Direct Seeding",
+    value: 2
+  },
+  {
+    title: "Assisted Natural Regeneration",
+    value: 3
+  }
+];
+const dropdownOptionsTarget = [
+  {
+    title: "Agroforest",
+    value: 1
+  },
+  {
+    title: "Natural Forest",
+    value: 2
+  },
+  {
+    title: "Mangrove",
+    value: 3
+  },
+  {
+    title: "Peatland",
+    value: 4
+  },
+  {
+    title: "Riparian Area or Wetland",
+    value: 5
+  },
+  {
+    title: "Silvopasture",
+    value: 6
+  },
+  {
+    title: "Woodlot or Plantation",
+    value: 7
+  },
+  {
+    title: "Urban Forest",
+    value: 8
+  }
+];
+
+const dropdownOptionsTree = [
+  {
+    title: "Single Line",
+    value: 1
+  },
+  {
+    title: "Partial",
+    value: 2
+  },
+  {
+    title: "Full",
+    value: 3
+  }
+];
+const AttributeInformation = () => {
+  return (
+    <div className="flex flex-col gap-4">
+      <Input
+        label="Polygon Name"
```

Comment:
Please make sure all free text in this component is wrapped in `t()`

Reply:
done